File: typescript/PG_TEST/fizzBuzz/fizzBuzz.ts
function pgt2(arg: number) {
  let result = '';
  if (arg % 5 === 0 && arg % 3 === 0) {
    result = 'FizzBuzz';
  } else if (arg % 3 === 0) {
    result = 'FIzz';
  } else if (arg % 5 === 0) {
    result = 'Buzz';
  } else {
    result = String(arg);
  }
  return result;
}
console.log(pgt2(6));

File: rust/rustbook/ch1/rust/src/fizzbuzz.rs
fn main() {
    //1から100まで繰り返す
    // for i in 1..101 {
    //     //条件を一つづつ判定する
    //     if i % 3 == 0 && i % 5 == 0 {
    //         println!("FizzBuzz");
    //     } else if i % 3 == 0 {
    //         println!("Fizz");
    //     } else if i % 5 == 0 {
    //         println!("Buzz");
    //     } else {
    //         println!("{}", i);
    //     }
    // }
    for i in 1..101 {
        if i % 3 == 0 && i % 5 == 0 {
            println!("FizzBuzz");
        } else if i % 3 == 0 {
            println!("Fizz");
        } else if i % 5 == 0 {
            println!("Buzz");
        } else {
            println!("{}",i);
        }
    }
}
